hdfs_namenode/datadog_checks/hdfs_namenode/data/conf.yaml.example

```diff
@@ -4,8 +4,8 @@ instances:
 
     ## @param hdfs_namenode_jmx_uri - string - required
     ## The HDFS DataNode check retrieves metrics from the HDFS DataNode's JMX
-    ## interface. This check must be installed on a HDFS DataNode. The HDFS
-    ## DataNode JMX URI is composed of the DataNode's hostname and port.
+    ## interface via HTTP(S) (not a JMX remote connection). This check must be installed on 
+    ## a HDFS DataNode. The HDFS DataNode JMX URI is composed of the DataNode's hostname and port.
```

Comment:
Shouldn't this be HDFS Namenode everywhere in that doc ? Also I think like the same update needs to be done for HDFS DataNode integration

Reply:
Finally got around to fixing the naming. I've also made a [PR](https://github.com/DataDog/integrations-core/pull/3963) for documenting the https port requirement for the HDFS DataNode integration.